content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -91,6 +91,21 @@ kubeadm to tell it what to do. This crashloop is expected and normal.
 After you initialize your control-plane, the kubelet runs normally.
 {{< /note >}}
 
+Add an local DNS alias for our master server. Edit the /etc/hosts file and add the above IP address and assign a
+name k8smaster.
+root@lfs458-node-1a0a:~# vim /etc/hosts
```

Comment:
we discussed that we don't want to have /etc/hosts examples in our docs.
this section should suffice:
https://github.com/kubernetes/website/blob/48af15ad2106bb55c61e51482ba6b6b854333dc5/content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md#considerations-about-apiserver-advertise-address-and-controlplaneendpoint

as far as the ClusterConfiguration example goes, it feels too generic to include.

this section includes more complete documentation examples:
https://github.com/kubernetes/website/blob/48af15ad2106bb55c61e51482ba6b6b854333dc5/content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md#more-information